plugins/amd-kria/fu-amd-kria-plugin.c

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2024 Advanced Micro Devices Inc.
+ * All rights reserved.
+ *
+ * This file is provided under a dual MIT/LGPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ * AMD Chooses the MIT license part of Dual MIT/LGPLv2 license agreement.
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR MIT
+ */
+
+#include "config.h"
+
+#include "fu-amd-kria-plugin.h"
+#include "fu-amd-kria-device.h"
+#include "fu-amd-kria-image-reg.h"
+#include "fu-amd-kria-persistent-reg.h"
+
+struct _FuAmdKriaPlugin {
+	FuPlugin parent_instance;
+};
+
+G_DEFINE_TYPE(FuAmdKriaPlugin, fu_amd_kria_plugin, FU_TYPE_PLUGIN)
+
+static void
+fu_amd_kria_plugin_init(FuAmdKriaPlugin *self)
+{
+}
+
+static gboolean
+fu_amd_kria_plugin_process_image(FuPlugin *plugin, FuDevice *dev, GError **error)
+{
+	FuDevice *system;
+	g_autoptr(GBytes) bytes = NULL;
+	g_autoptr(FuFirmware) firmware = NULL;
+	g_autoptr(FuProgress) progress = fu_progress_new(NULL);
+
+	bytes = fu_device_dump_firmware(dev, progress, error);
+	if (bytes == NULL)
+		return FALSE;
+	firmware = fu_amd_kria_image_firmware_new();
+
+	if (!fu_firmware_parse(firmware, bytes, 0x0, error))
```

Comment:
I think you want `FWUPD_INSTALL_FLAG_NONE` here

Reply:
Ack